lms/tasks/email_digests.py

```diff
@@ -91,13 +90,6 @@ def send_instructor_email_digest_tasks(*, batch_size):
                         select(candidate_courses.c.course_id)
                     ),
                     LTIRole.type == "instructor",
-                    # No EmailUnsubscribes
-                    User.h_userid.not_in(
-                        select(EmailUnsubscribe.h_userid).where(
-                            EmailUnsubscribe.tag
-                            == EmailUnsubscribe.Tag.INSTRUCTOR_DIGEST
-                        )
-                    ),
```

Comment:
The `not_()` below filters out users based on the `user_preferences` table instead